src/components/SingleClub/index.jsx

```diff
@@ -43,10 +43,10 @@ export default function SingleClub({ club }) {
         </div>
 
         <div id={`less${club._id}`} className="desc">
-          {club.description.length > 80
-            ? club.description.slice(0, 80) + '...'
-            : club.description}
-          {club.description.length > 80 ? (
+          {club.tagLine.length > 40
```

Comment:
Can we remove this ? 
Like we cannot technically input a tagline with more than 50 characters. So i think maybe the expand feature is not of much use in this scenario

Reply:
changes done ✅✅✅✅✅